src/dune_engine/package.ml

```diff
@@ -431,6 +433,22 @@ module Info = struct
     ; maintainers = None
     }
 
+  let default =
+    { source =
+        Some
+          (Host
+             { kind = Source_kind.Host.Github
+             ; user = "username"
+             ; repo = "reponame"
+             })
+    ; license = Some "LICENSE"
+    ; authors = Some [ "Author Name" ]
+    ; homepage = Some "https://url/to/home/page"
+    ; bug_reports = Some "https://url/to/issue/tracker"
+    ; documentation = Some "https://url/to/documentation"
+    ; maintainers = Some [ "Maintainer Name" ]
```

Comment:
Cargo uses git to fill in the author name and email. I don't think that's worth the complication of shelling out to git, and the rigidity of tying initialization to a specific VCS. The point here is to give users a clear and easy template they can expand upon and fill in.

Reply:
If the source is `(github ...)`, these fields are automatically inferred. Given that, what about using `None` for these fields?

Reply:
Yeah, good idea!

See 8b718b133dc6087b75fb12018bdfc4fdec813051